CmisSync.Lib/Sync/CrawlStrategy.cs

```diff
@@ -220,7 +220,7 @@ private void CrawlRemote(IFolder remoteFolder, string localFolder, IList<string>
                     }
                     else
                     {
-                        Logger.Debug("Unknown object type: " + cmisObject.ObjectType.DisplayName
+                        Logger.Warn("Unknown object type: " + cmisObject.ObjectType.DisplayName
```

Comment:
Is this necessary? This will generate too many warnings in repositories that contain a lot of exotic types.